xml/common_intro_feedback_i.xml

```diff
@@ -75,11 +83,31 @@
    <term>Mail</term>
    <listitem>
     <para>
-     Alternatively, you can report errors and send feedback concerning the
-     documentation to <email>doc-team@suse.com</email>. Make sure to include
-     the document title, the product version and the publication date of the
-     documentation. Refer to the relevant section number and title (or include
-     the URL) and provide a concise description of the problem.
+     You can also report errors and send feedback concerning the
+     documentation to <email>doc-team@suse.com</email>. Include the
+     document title, the product version, and the publication date of the
+     document. Additionally, include the relevant section number and title (or
+     provide the URL) and provide a concise description of the problem.
+    </para>
+   </listitem>
+  </varlistentry>
+  <varlistentry condition="beta">
```

Comment:
I'm not sure this profiling condition would work out of the box, let me check tomorrow.

Reply:
The text appears but as we don't use the related profiling parameter in the DC file, it will always appear, irrelevant if the beta phase is running or not. I would therefore remove it from here and keep it in doc-kit.